backend/internal/model/game_test.go

```diff
@@ -4,52 +4,55 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	"errors"
 )
 
 func TestResetScore(t *testing.T) {
-	gS := &gameScore{3, 1}
+	game := &Game{gameScore{3, 1}}
+
 	type args struct {
-		score *gameScore
+		name               string
+		expectedBlueScore  int
+		expectedWhiteScore int
 	}
-	tests := []struct {
-		name string
-		args args
-	}{
-		{"Check if values are set to 0", args{gS}},
+
+	tests := []args{
+		{name: "Check if values are set to 0", expectedBlueScore: 0, expectedWhiteScore: 0},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			gS.ResetScore()
+			game.ResetScore()
 		})
-		if tt.args.score.WhiteScore != 0 || tt.args.score.BlueScore != 0 {
-			t.Errorf("Score did not reset. Goals white: %v, Goals blue: %v", tt.args.score.WhiteScore, tt.args.score.BlueScore)
+		if tt.expectedWhiteScore != 0 || tt.expectedBlueScore != 0 {
+			t.Errorf("Score did not reset. Goals white: %v, Goals blue: %v", tt.expectedWhiteScore, tt.expectedBlueScore)
 		}
```

Comment:
Fancy setup isn't needed for only 1 test case

Reply:
We are aware that it is unnecessary but for now we will leave it as it is.